src/content/en/updates/2017/06/credential-management-updates.md

```diff
@@ -0,0 +1,333 @@
+
+        return res.json();
+
+      } else {
+
+        throw 'Auth failed';
+
+      }
+
+    }).then(profile => {
+
+      console.log('Auth succeeded', profile);
+
+    });
+
+### Custom fetch will be deprecated soon
+
+Now that passwords return in the `PasswordCredential` object,
+the custom `fetch()` function will stop functioning starting Chrome 62.
+All developers **MUST** make relevant changes to not use a custom `fetch()`.
+
+To determine if you are using a custom `fetch()` function,
+check if it uses a `PasswordCredential` object or a `FederatedCredential` object
+as a value of credentials property and remove it, for example:
+
+    fetch('/signin', {
+
+      method: 'POST',
+
+      **credentials: c**
+
+    })
+
+### `navigator.credentials.get()` now accepts an enum mediation
+
+Until Chrome 60,
+`navigator.credentials.get()` accepted an optional `unmediated` property
+with a boolean flag. For example:
+
+    navigator.credentials.get({
+
+      password: true,
+
+      federated: {
+
+        provider: [ 'https://accounts.google.com' ]
+
+      },
+
+      unmediated: true
+
+    }).then(c => {
+
+      // Sign-in
+
+    });
+
+This positive flag instructs the browser to skip a user mediation.
+The user mediation could happen when:
+
+* A user needs to choose an account to sign-in with.
+
+* A user wants to explicitly sign-in
+after `navigator.credentials.requireUseMediation()` call.
+
+The flag is now extended as mediation, for example:
+
+    navigator.credentials.get({
+
+      password: true,
+
+      federated: {
+
+        provider: [ 'https://accounts.google.com' ]
+
+      },
+
+      mediation: 'silent'
+
+    }).then(c => {
+
+      // Sign-in
+
+    });
+
+Choose one of the following options for the `mediation` value:
+
+<table>
+  <tr>
+    <th><code>mediation</code> value</th>
+    <th>Compared to boolean flag</th>
+    <th>Behavior</th>
+  </tr>
+  <tr>
+    <td><code>silent</code></td>
+    <td>Equals <code>unmediated: true</code></td>
+    <td>Credential passed without showing an account chooser.</td>
+  </tr>
+  <tr>
+    <td><code>optional</code></td>
+    <td>Equals <code>unmediated: false</code></td>
+    <td>Shows an account chooser if
+    <code>preventSilentAccess()</code> called previously.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>required</code></td>
+    <td>A new option</td>
+    <td>Always show an account chooser.
+    Useful when you want to let a user to switch an account
+    using the native account chooser dialog.</td>
+  </tr>
+</table>
+
+### `requireUserMediation()` renamed to `preventSilentAccess()`
+
+To align nicely with the new mediation property offered in the `get()` call ,
+the `navigator.credentials.requireUserMediation()` method has been renamed to
+`navigator.credentials.preventSilentAccess()`. 
+
+See the previous table
+to understand how calling this function affects `navigator.credentials.get()`.
+
+The renamed method prevents passing a credential
```

Comment:
I'd rewrite as: `The renamed method prevents passing a credential without showing the account chooser (sometimes called without user mediation).`